pages/school-administrators.md

```diff
@@ -601,9 +582,9 @@ relatedlinks:
 </va-accordion>
 <section class="merger-majorlinks va-nav-linkslist va-nav-linkslist--related">
   <section class="field_related_links">
-    <h2 class="va-nav-linkslist-heading" tabindex="-1">
+    <h3 class="va-nav-linkslist-heading" tabindex="-1">
```

Comment:
This section is a subsection of the prior section, so the heading level is increased.